.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,7 +27,25 @@
 - [ ] Yes
 - [ ] No
 
+#### Has This Been Tested?
+
+- [ ] Yes
+- [ ] No
+
 ### Other information
+Target: 
+```
+
+```
+
+#### How Has This Been Tested?
```

Comment:
@TommasoLencioni Tests should exist somewhere, wouldn't it be nice to provide a link?